distribution/openrct2.d.ts

```diff
@@ -172,6 +172,12 @@ declare global {
      * Core APIs for storage and subscriptions.
      */
     interface Context {
+
+        /**
+         * Gets all of the currently registered plugins
+         */
+        allPlugins(): PluginMetadata[];
```

Comment:
```suggestion
        getPlugins(): PluginMetadata[];
```
Most of the scripting API has get/set